### PR TITLE
Leaderboard-project

### PR DESCRIPTION
**Pull request by elisha2kyakpo1**
This is a leaderboard project milestone1.

- I created an HTML mark-up.
- I styled the elements.
- I created few functions to add and display user inputs from the form.
- I configured webpack

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @elisha2kyakpo1, 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._
- [ ] Kindly update the readme adding certain necessary sections.

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```
Comment (mikenath223) [inline dist/index.html:1]: - [ ] In the `readme.md` there are some necessary sections that are lacking. Kindly follow [this template](https://github.com/microverseinc/readme-template) and include them.

**Comment by elisha2kyakpo1** [comment]
Thanks, @mikenath223
I don't know how I forgot this.

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**